take-homes/fe-take-home.md

```diff
@@ -0,0 +1,29 @@
+# NY Times News Reader Take Home Test
+
+You will create a news reader application that pulls articles from the New York Times API.
+
+## Requirements:
+
+At a minimum, you must show:
+
+* A list of articles 
```

Comment:
Do you think a minimum/maximum list should be included here or whether or not 'pagination' should be used?

Reply:
For that API I think it's usually less than 100 stories per call. I'm inclined to leave it vague where they could paginate it if they wanted 🤷 . 

One thing I like about this prompt is how it makes them think with a user focus and what is necessary for their less than 6 hour mvp.

@Kalikoze what do you think of the scope of this. If you think it's light for 6 hours we could always add some more requirements too like favoriting or filtering by a tag

Reply:
I'm fine with the scope of this for now.  This gives them the bare minimum, but I do think it gives them an opportunity to do something to make it stand out whether it's testing or accessibility or adding a new feature.  I'm inclined to try less is more for now.  We can try it out this time and if it feels way too small, we can always add on in our next iteration.